README.md

```diff
@@ -1,2 +1,78 @@
 # rust
-Docker image for Rust
+
+This is a Docker image with Rust 1.18 installed.
+
+## How to Use
+
+In order to use this image for your project, edit the `techio.yml` file and add the following lines to your project:
+
+```yaml
+    runner: techio/rust:latest
+```
```

Comment:
Added in https://github.com/CodinGame/rust-template/pull/1/commits/b530d6628d4ef93ee88b4b086fe3e488e483528c